lib/rpm_database.rb

```diff
@@ -132,11 +133,15 @@ def parse_stat_line(line)
       "dir"
     when "symbolic link"
       "link"
-    when "regular file"
+    when / file$/
```

Comment:
We already have two methods just for stat line so I would prefer not to change it.